package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ionic/angular-toolkit",
-  "version": "1.5.1",
+  "version": "2.0.0",
```

Comment:
Please let semantic-release dictate version numbers. We'll use `BREAKING CHANGE:` in the commit message for this PR. https://www.conventionalcommits.org/en/v1.0.0-beta.4/